### PR TITLE
Introduces autofocus of password when remember user is checked

### LINKED ISSUE #2954: Move cursor to `Password` field when `Remember User` was selected
When `Remember User` selected, activate cursor in `Password` to save a click and enter password with out moving courser(press tab) manually.
Screen Shot 2021-05-23 at 2 17 36 AM

### PR DESCRIPTION
**Pull request by kelsos**
Closes #2954

### CODE REVIEW

File: frontend/app/src/components/account-management/Login.vue [hunk 1]
```diff
@@ -316,6 +317,19 @@ export default class Login extends Vue {
...
+    }
+  }
+
+  private focusPassword() {
+    this.$nextTick(() =>
+      setTimeout(() => {
+        (this.$refs.password as any).$el
+          .querySelector('input:not([type=hidden])')
+          .focus();
+      }, 100)
```
Comment (kelsos) [inline frontend/app/src/components/account-management/Login.vue:331]: This was a bit more complicated than I expected. I had to go through a couple of so threads.

**Comment by yabirgb** [comment]
I get this error and the password field wasn't autofocused

image

**Comment by kelsos** [comment]
interesting.

**Comment by kelsos** [comment]
Ok, it seems that this happens on a fresh start but not when refreshing.

**Comment by kelsos** [comment]
To test the focus working properly in dev mode you have to comment the dev tools opening. Otherwise it will mess with the focus
https://github.com/rotki/rotki/blob/2ce27ea44960a3c0ba87c3910d2fcf77052f45dd/frontend/app/src/background.ts#L109

**Comment by yabirgb** [comment]
Okey for me it's fine. As I understand this will work correctly on the final release so we can merge it :)

**Review by yabirgb — APPROVED** [review]

**Pull request merged**